packages/zipkin-instrumentation-axiosjs/test/integrationTest.js

```diff
@@ -1,408 +1,214 @@
-
-        expect(annotations[4].annotation.annotationType).to.equal('ServerAddr');
-
-        expect(annotations[5].annotation.annotationType).to.equal('BinaryAnnotation');
-        expect(annotations[5].annotation.key).to.equal('http.status_code');
-        expect(annotations[5].annotation.value).to.equal('404');
-
-        expect(annotations[6].annotation.key).to.equal('error');
-        expect(annotations[6].annotation.value).to.equal('404');
-
-        expect(annotations[7].annotation.annotationType).to.equal('ClientRecv');
-
-        expect(annotations[8]).to.be.undefined; // eslint-disable-line no-unused-expressions
-
+  it('should report 400 in tags', done => {
+    const badPath = '/weather/securedTown';
+    getClient()({url: `${baseUrl}${badPath}`, timeout: 100})
+      .catch(error => {
+        verifyGetSpan({
+          'http.path': badPath,
+          'http.status_code': '400',
+          'error': '400'
+        });
         done();
       });
-    });
   });
 
-  it('should report when service does not exist', done => {
-    tracer.scoped(() => {
-      const zipkinAxiosClient = getClient();
-      const host = 'localhost:12345';
-      const url = `http://${host}`;
-      zipkinAxiosClient({url, timeout: 200}).catch(() => {
-        const annotations = record.args.map(args => args[0]);
-        const initialTraceId = annotations[0].traceId.traceId;
-        annotations.forEach(ann => expect(ann.traceId.traceId)
-            .to.equal(initialTraceId).and
-            .to.have.lengthOf(16));
-
-        expect(annotations[0].annotation.annotationType).to.equal('ServiceName');
-        expect(annotations[0].annotation.serviceName).to.equal('weather-app');
-
-        expect(annotations[1].annotation.annotationType).to.equal('Rpc');
-        expect(annotations[1].annotation.name).to.equal('GET');
-
-        expect(annotations[2].annotation.annotationType).to.equal('BinaryAnnotation');
-        expect(annotations[2].annotation.key).to.equal('http.path');
-        expect(annotations[2].annotation.value).to.equal('/');
-
-        expect(annotations[3].annotation.annotationType).to.equal('ClientSend');
-
-        expect(annotations[4].annotation.annotationType).to.equal('ServerAddr');
-
-        expect(annotations[5].annotation.annotationType).to.equal('BinaryAnnotation');
-        expect(annotations[5].annotation.key).to.equal('error');
-        expect(annotations[5].annotation.value)
-            .to.contain('Error: connect ECONNREFUSED 127.0.0.1:12345');
-
-        expect(annotations[6].annotation.annotationType).to.equal('ClientRecv');
-
-        expect(annotations[7]).to.be.undefined; // eslint-disable-line no-unused-expressions
+  it('should report 500 in tags', done => {
+    const badPath = '/weather/bagCity';
+    getClient()({url: `${baseUrl}${badPath}`, timeout: 100})
+      .catch(error => {
+        verifyGetSpan({
+          'http.path': badPath,
+          'http.status_code': '500',
+          'error': '500'
```

Comment:
this is a good example as it shows all tags succinctly are in the same span document